importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -0,0 +1,132 @@
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static io.sphere.sdk.client.SphereClientUtils.blockingWait;
+import static java.util.Arrays.asList;
+import static java.util.stream.Collectors.toList;
+
+@Component
+@Lazy
+public class AvailabilityJobConfiguration extends DefaultCommercetoolsJobConfiguration {
+    private static final Logger logger = LoggerFactory.getLogger(AvailabilityJobConfiguration.class);
+
+    @Bean
+    public Job productsSuggestionsCopyJob(final Step channelImportStep, final Step customTypeImportStep) {
+        return jobBuilderFactory.get("importJoyrideAvailabilityJob")
+                  .start(customTypeImportStep)
+                  .next(channelImportStep)
+                .build();
+    }
+
+    @Bean
+    public Step channelImportStep(final BlockingSphereClient sphereClient,
+                                  final ItemReader<ChannelDraftDsl> channelsDraftReader,
+                                  final ItemWriter<ChannelDraftDsl> channelsDraftItemWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("channelImportStep");
+        return stepBuilder
+                .<ChannelDraftDsl, ChannelDraftDsl>chunk(1)
+                .reader(channelsDraftReader)
+                .writer(channelsDraftWriter(sphereClient))
+                .build();
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<ChannelDraftDsl> channelsDraftReader(@Value("#{jobParameters['resource']}") final Resource channelsJsonResource) throws IOException {
+        logger.info("URL_Channels: " + channelsJsonResource);
+        final ObjectReader reader = SphereJsonUtils.newObjectMapper().readerFor(new TypeReference<List<ChannelDraftDsl>>() {
+        });
+        final InputStream inputStream = channelsJsonResource.getInputStream();
+        final List<ChannelDraftDsl> list = reader.readValue(inputStream);
+        return new ListItemReader<>(list);
+    }
+
+    @Bean
+    protected ItemWriter<ChannelDraftDsl> channelsDraftWriter(final BlockingSphereClient sphereClient) {
+        return items -> items.forEach(channelDraft -> sphereClient.executeBlocking(ChannelCreateCommand.of(channelDraft)));
+    }
+
+    @Bean
+    public Step customTypeImportStep(final BlockingSphereClient sphereClient,
+                                  final ItemReader<TypeDraft> customTypeReader,
+                                  final ItemWriter<TypeDraft> customTypeItemWriter) {
+        final StepBuilder stepBuilder = stepBuilderFactory.get("customTypesImportStep");
+        return stepBuilder
+                .<TypeDraft, TypeDraft>chunk(1)
+                .reader(customTypeReader)
+                .writer(customTypeWriter(sphereClient))
+                .build();
+    }
+
+    @Bean
+    @StepScope
+    private ItemReader<TypeDraft> customTypeReader(@Value("#{jobParameters['resource_types']}") final Resource typesJsonResource) throws IOException {
```

Comment:
do not use snake case here, in the other parameters we use CamelCase: https://github.com/commercetools/commercetools-sunrise-data/tree/master/importApp#prepare-a-payload-json-file